README.md

```diff
@@ -651,29 +651,73 @@ separate section above).
 *form file contracts before running the deploy script. Otherwise the health
 *checks will fail as your node is not ready to upload and download
 
-### Run Docker Command
+### Run Shell Commands
 
-Playbook:
+There are several options to run shell commands against portals via Ansible.
 
-- Run a docker command on portals define in `docker_commands` variable.
+Requires (all options):
 
-Preparation:  
-Defined a `docker_commands` variable in your `portal_versions.yml` file.
+- `shell_commands` variable must be defined as a list of commands to be run.
+- It can be defined either in `my-vars/portal_versions.yml` (e.g. for
+  `portals-deploy` playbook with other variables) or in a separate file e.g.
+  `my-vars/shell-commands.yml`
+
+Depreciation note:
+
+Previously there was `docker_commands` variable with the similar functionality,
+its usage is now depreciated, but for backwards compatibility it works as it
+used to work.
+
+If both `docker_commands` and `shell_commands` are used, commands in both of
+these lists are run (commands from these two lists are added together).
 
 Example:
 
 ```yaml
 ---
-docker_commands:
+shell_commands:
   - "docker exec sia siac"
   - "docker exec sia siac renter"
 ```
 
+Wrapping commands into quotes is optional, but might be needed if Ansible/Yaml
+syntax requires it.
+
+#### Disable from a load balancer, run shell commands, test and enable portals
+
+See above for `shell_commands` configuration.
+
+Playbook:
+
+- Disables health check.
+- Runs shell commands defined in `shell_commands` variable and logs outputs.
+- Runs integration tests and health checks.
+- Enables portals.
+
 To run:  
-`scripts/portals-docker-command.sh -e @my-vars/config.yml --limit eu-fin-1`
 
-The deploy script also supports the docker command execution:
-`scripts/portals-deploy.sh -e @my-vars/config.yml --limit eu-fin-1`
+`scripts/portals-run-shell-commands-with-disable-run-test-enable.sh -e @my-vars/shell-commands.yml --limit eu-fin-1`
```

Comment:
nit:

```suggestion
`scripts/portals-run-shell-commands.sh -e @my-vars/shell-commands.yml --limit eu-fin-1`
```

With my other comment about not including the option to run these without disabling the LB, this should be simplified.